crates/solidity/inputs/language/bindings/rules.msgb

```diff
@@ -2137,6 +2137,28 @@ inherit .star_extension
   edge @enum.def -> type
   edge type -> type_enum_type
   edge type_enum_type -> @enum.lexical_scope
+
+  ; Path to link min and max to the enum's scope (#1158)
+  ; It resolves paths of the form `type(<enum>).min.<something>`
+  node typeof
+  attr (typeof) pop_symbol = "@typeof"
+
+  node built_in_member
+  attr (built_in_member) pop_symbol = "."
+  node min_built_in
+  attr (min_built_in) pop_symbol = "min"
+  node max_built_in
+  attr (max_built_in) pop_symbol = "max"
+  node min_max_accessor
+  attr (min_max_accessor) pop_symbol = "."
+
+  edge type -> typeof
+  edge typeof -> built_in_member
+  edge built_in_member -> min_built_in
+  edge built_in_member -> max_built_in
+  edge min_built_in -> min_max_accessor
+  edge max_built_in -> min_max_accessor
+  edge min_max_accessor -> @enum.lexical_scope
```

Comment:
This is not correct. The trailing `.` in the path (the `min_max_accessor` node) allows binding `min.` with anything in the lexical scope, which is why the test case resolves correctly. But it would also bind to any symbol at that scope.

Keep in mind you should "replace" (as in the symbol stack, pop and then push) `min` and `max` with a `@typeof -> X`, where `X` is the argument to `type()` so that you can continue the binding resolution at `X`.